src/actions/password.reset.action.js

```diff
@@ -0,0 +1,17 @@
+import axios from 'axios';
+import { ERROR_ACTION, RESET_PASSWORD_ACTION } from '../constants';
+
+const resetPasswordRequest = ({ email }) => dispatch => axios.post('http://127.0.0.1:8000/api/users/email_sent', { email })
```

Comment:
Have this part of the url as an environment variable like so:
`REACT_APP_BASE_URL=http://127.0.0.1:8000/api/`